extra-utils/direnv/spec

```diff
@@ -1,3 +1,3 @@
-VER=2.23.1
+VER=2.24.0
 SRCTBL="https://github.com/direnv/direnv/archive/v$VER.tar.gz"
```

Comment:
Please use the new ACBS [specification file format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/#spec).